packages/framework/src/Model/Order/OrderFacade.php

```diff
@@ -158,6 +158,10 @@ public function edit(int $orderId, OrderData $orderData): Order
         $orderTotalPrice = $this->orderPriceCalculation->getOrderTotalPrice($order);
         $order->setTotalPrices($orderTotalPrice->getPrice(), $orderTotalPrice->getProductPrice());
 
+        if ($orderData->withdrawalRequestData !== null && $order->getWithdrawalRequest() !== null) {
+            $order->getWithdrawalRequest()->edit($orderData->withdrawalRequestData);
```

Comment:
When I call 

```
$orderData = $this->orderDataFactory->createFromOrder($order);

$orderData->withdrawalRequestData = null;

// edit/flush
```

The result is unexpected - it's ignored instead of set to null